inc/config.class.php

```diff
@@ -14,51 +15,95 @@ class PluginSatisfactionsmileyConfig extends CommonDBTM {
     * @param integer $nb number of elements
     * @return string name of this type
     */
-   static function getTypeName($nb = 0) {
+   static function getTypeName($nb = 0)
+   {
       return __('Configuration', 'satisfactionsmiley');
    }
 
-   function showForm($id, $options = []) {
+   function showForm($id, $options = [])
+   {
 
       $this->getFromDB($id);
       $this->initForm($id, $options);
       $this->showFormHeader($options);
+      $checked = '';
 
       echo "<tr class='tab_bg_1'>";
-      echo "<td>".__('Smiley very bad', 'satisfactionsmiley')."</td>";
-      echo "<td><img src='".$this->fields['smiley_1']."'></td>";
+      echo "<td>";
+      if($this->fields['is_active_1'] == 1) {
+         $checked = 'checked';
+      } else {
+         $checked = '';
+      }
+      echo "<input type='checkbox' name='check_list[]' value='1' $checked>";
+      echo "</td>";
+      echo "<td>" . __('Smiley very bad', 'satisfactionsmiley') . "</td>";
+      echo "<td><img src='" . $this->fields['smiley_1'] . "'></td>";
       echo "<td colspan='2'>";
```

Comment:
in GLPI, in form by default there are 4 columns, so 4 TD, remove the ` colspan='2'` to be 1 col instead 2 and you will have 4 `<td>`